.github/workflows/ci.yml

```diff
@@ -55,5 +55,5 @@ jobs:
       - name: Ensure repo README is up-to-date
         run: node repo-scripts/repo-toolbox/lib/start.js readme --verify
 
-      - name: Rush test (rush-lib)
-        run: node apps/rush/lib/start-dev.js test --verbose --production --timeline
+      # - name: Rush test (rush-lib)
+      #  run: node apps/rush/lib/start-dev.js test --verbose --production --timeline
```

Comment:
As discussed with @octogonz offline, we may want to comment out this line temporary as it is not a correct step up. 

Here is the problem: 
For this `ci.yaml` file, 
Line 46, it uses the rush version defined in rush.json to perform the install
Line 53, it uses the rush version defined in rush.json to perform the build and test

Line 59, it uses the rush version from rush-lib folder to perform the build and test

If I understand correctly here, Line 59 reuses the rush install results from line 46. However, the rush versions used on line 46 and line 59 **are different** 

This causes CI check failure when updating `pnpm-sync-lib` version. 
In this PR, we updated the `pnpm-sync-lib` version. 
So, in the above CI configuration, Line 46, when it completes, it generates a `pnpm-sync.json` based the `pnpm-sync-lib` version bundled in rush version defined in rush.json. 

Line 59, since we updated the `pnpm-sync-lib` version in this PR,  when **using rush version from rush-lib (which has a updated pnpm-sync-lib)** folder to perform the build and test, it will throw an error saying `pnpm-sync.json` version is out of date!
It is due to `Rush test (rush-lib)` step reuses the installation from previous step, but the rush version used in the previous step is not same as this step.

Reply:
We will reenable this after the PR is merged.

However @iclanton I think  it is fundamentally incorrect for this CI pipeline to invoke `rush test` using a different version of Rush than `rush install`.

Reply:
@g-chao fixed it by adding a separate `rush install` step